modules/ROOT/pages/index.adoc

```diff
@@ -8,26 +15,129 @@
+* xref:clustering/servers.adoc[Manage servers in a cluster]
+* xref:clustering/databases.adoc[Manage databases in a cluster]
+* xref:authentication-authorization/manage-users.adoc[Manage users]
+* xref:authentication-authorization/manage-roles.adoc[Manage roles]
+* xref:authentication-authorization/manage-privileges.adoc[Manage privileges]
+* xref:authentication-authorization/built-in-roles.adoc[Built-in roles and privileges]
+* xref:authentication-authorization/immutable-roles-privileges.adoc[Manage immutable roles and privileges]
+
+
+[.widget.lists]
+== Tailored for you
+
+=== Neo4j Admin dbms commands
+
+[.icon]
+image:icon-admin.svg[]
+
+[.list]
+* xref:configuration/set-initial-password.adoc[`neo4j-admin dbms set-initial-password`]
+* `neo4j-admin dbms set-default-admin`
+* `neo4j-admin dbms unbind-system-db`
+
+=== Neo4j Admin server commands
+
+[.icon]
+image:icon-server.svg[]
+
+[.list]
+* `neo4j-admin server console`
+* `neo4j-admin server get-id`
+* `neo4j-admin server license`
+* xref:tools/neo4j-admin/neo4j-admin-memrec.adoc[`neo4j-admin server memory-recommendation`]
+* xref:tools/neo4j-admin/migrate-configuration.adoc[`neo4j-admin server migrate-configuration`]
+* xref:tools/neo4j-admin/neo4j-admin-report.adoc[`neo4j-admin server report`]
+* `neo4j-admin server restart`
+* `neo4j-admin server start`
+* `neo4j-admin server status`
+* `neo4j-admin server stop`
+* xref:tools/neo4j-admin/unbind.adoc[`neo4j-admin server unbind`]
+* xref:/tools/neo4j-admin/validate-config.adoc[`neo4j-admin server validate-config`]
+* `neo4j-admin server windows-service`
+
+=== Neo4j Admin database commands
+
+[.icon]
+image:icon-dbsettings.svg[]
+
+[.list]
+* xref:tools/neo4j-admin/consistency-checker.adoc[`neo4j-admin database check`]
+* xref:tools/neo4j-admin/neo4j-admin-store-info.adoc[`neo4j-admin database info`]
+* xref:tools/neo4j-admin/neo4j-admin-import.adoc[`neo4j-admin database import`]
+* xref:tools/neo4j-admin/upload-to-aura.adoc[`neo4j-admin database upload`]
+* xref:tools/neo4j-admin/migrate-database.adoc[`neo4j-admin database migrate`]
+* xref:backup-restore/copy-database.adoc[`neo4j-admin database copy`]
+
+=== Neo4j Admin database backup commands
 
-ifdef::backend-pdf[]
-include::license.adoc[leveloffset=+1]
-endif::[]
+[.icon]
+image:icon-import.svg[]
 
-ifdef::backend-html5[(C) {copyright}]
-ifndef::backend-pdf[]
+[.list]
+* xref:backup-restore/online-backup.adoc[`neo4j-admin database backup`]
+* xref:backup-restore/aggregate.adoc[`neo4j-admin backup aggregate`]
+* xref:backup-restore/inspect.adoc[`neo4j-admin backup inspect`]
+* xref:backup-restore/restore-backup.adoc[`neo4j-admin database restore`]
+* xref:backup-restore/offline-backup.adoc[`neo4j-admin database dump`]
+* xref:backup-restore/restore-dump.adoc[`neo4j-admin database load`]
 
-Documentation license: link:{common-license-page-uri}[Creative Commons 4.0]
-endif::[]
-ifdef::backend-pdf[]
-(C) {copyright}
+[.next-steps]
```

Comment:
Not sure if we need that. What does it mean?

Reply:
Just CSS for the component to work :)

Reply:
Can we just have a sentence with no bullet or heading?

Reply:
No, that's how it is in all subpages.